lms/product/family.py

```diff
@@ -0,0 +1,18 @@
+from enum import Enum
+
+
+class Family(str, Enum):
```

Comment:
Make this its own module, much easier to import without creating circular imports.